pkg/products/rhsso/reconciler.go

```diff
@@ -48,9 +48,10 @@ var (
 )
 
 const (
-	SSOLabelKey   = "sso"
-	SSOLabelValue = "integreatly"
-	RHSSOProfile  = "RHSSO"
+	SSOLabelKey               = "sso"
+	SSOLabelValue             = "integreatly"
+	RHSSOProfile              = "RHSSO"
+	adminCredentialSecretName = "credential-rhsso"
```

Comment:
This needs to be dynamic, based on `credential-` + keycloakName var from above

Reply:
Good catch ! Will make it combination based on the `keyCloakName` var